bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -151,6 +154,44 @@ private boolean isTargetBranchAllowed() {
                 .filter(link -> link.relationship().isPresent() && "csr for".equals(link.relationship().get())).findAny();
     }
 
+    private Optional<Issue> getJepIssue() {
+        var issueOpt = getJepIssueFromIssueTracker();
```

Comment:
Do we really need to get the issue from the issue tracker at this point? For other issues, CheckRun just gets the ID and description from the encoded marker pattern. See SolvesTracker.java.

Reply:
Fixed.